### PR TITLE
Fix babel proposal-numeric-separator issue

### PR DESCRIPTION
**Pull request by SuperS1018**
The problem is on babel side so the fix is to force to use lower version of babel. I also modify the instruction on the README.

### CODE REVIEW

File: chat/README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 # Chatroom
 This example demonstrates how to simply create a chat application with system managed state.
+
```
Comment (joshuaauerbachwatson) [inline chat/README.md]: I do not think this insertion should be here; the purpose of all of our demos is to motivate simple install with `nim deploy project`.

Comment (joshuaauerbachwatson) [reply chat/README.md]: If there is.a change to the build procedure, that should go in `web/build.sh` and `web/build.cmd`.  The user can read those to understand what the build consists of.

**Comment by joshuaauerbachwatson** [comment]
There is new news:  it appears that this problem got fixed upstream.   I am not actually surprised that this happened because, if the problem is in something that react tools are depending on there would be a large community squawking about it.

Anyway, @SuperS1018 , I recommend we just leave this PR open a bit longer (not really necessary to make my recommended changes).   We can close it without merging in a couple of days once it is really clear that the problem is gone.

**Comment by rabbah** [comment]
Closed as stale.

**Pull request closed without merging**